### PR TITLE
small sugestion

### PR DESCRIPTION
**Pull request by MatejMagat305**
well I don't know whether you want to video link in read me or not, but you decide ...

### CODE REVIEW

File: android/ndk/ndk.v [hunk 1]
```diff
@@ -226,7 +226,13 @@ pub fn host_arch() string {
 	} $else $if windows {
 		return 'windows-x86_64'
 	} $else $if termux {
-		return 'linux-aarch64'
+		$if arm64 {
```
Comment (Larpon) [inline android/ndk/ndk.v:229]: Please stop changing unrelated code in your pull requests, and submit collections of concrete changes instead. This PR is about adding a link. Please remove this change

Comment (MatejMagat305) [reply android/ndk/ndk.v]: sorry

**Comment by Larpon** [comment]
Thanks! This is super cool :sunglasses:

File: android/ndk/ndk.v (hunk 1 above)
Comment (Larpon) [reply android/ndk/ndk.v]: Do not be sorry, it's a learning process.

We ask this to keep everything as easy and clear as possible for all parts involved. It's easier to review and comment on specific changes - than multiple things.

If you like to change this part of the code just do it in a separate pull request and we're happy to have a look at it.
